packages/node_modules/@webex/react-component-activity-share-files/src/styles.css

```diff
@@ -7,6 +7,65 @@
   width: 320px;
 }
 
+@media (max-width: 320px) {
+  .shareList {
+    max-width: 100%;
+    width: unset;
+  }
+
+  .shareList div[class*="md-content__chat-inner-container"] {
+    width: unset;
+    min-width: 170px;
+  }
+
+  .shareList div[class=" md-content-file--full"] {
```

Comment:
Why do we have a space here?
```suggestion
  .shareList div[class="md-content-file--full"] {
```

Reply:
It's an exact match only if there is a preceding space
<img width="294" alt="image" src="https://github.com/user-attachments/assets/b52f157b-0fc4-4acc-8ef0-58af6bf89bae">